### ISSUE TITLE
About the experiment results

### ISSUE THREAD

**Issue by jmjff** [ATR-DBI/CityRefer#15]
Hello! Thank you for your wonderful work！
 I notice that in your readme ,when input is xyz+color+landmark, the accuracy(test) is 0.320. But when I use your pre-trained model or train from the scratch, the result is about 0.272 , lower than your checkpoints. 
![image](https://github.com/ATR-DBI/CityRefer/assets/129597090/e61be72f-e513-4142-acac-fbf2d19f068e)
So I wonder whether you are using a validation set or a test set to evaluate your model? 
Many thanks for your time.
Regards.

**Comment by miyatai2** [comment]
The accuracy(test) is the performance of the test set, not the val set.
The test set is not distributed.
